### PR TITLE
adding services pages

### LINKED ISSUE #8: Electrician services - Section

### PR DESCRIPTION
**Pull request by ksaini75**
Updated the Services Page, it is now responsive for both desktop and mobile platforms.

### CODE REVIEW

File: src/components/LandingPage/LandingPage.scss [hunk 1]
```diff
@@ -38,7 +38,8 @@ $btn-hover: #4a99c0;
 }
 
 .company-name {
-    font-size: 52px;
+    font-size: 52px !important;
```
Comment (MrGDS2) [inline src/components/LandingPage/LandingPage.scss:41]: Why is this changed to important?

File: src/components/LandingPage/LandingPage.scss [hunk 2]
```diff
@@ -115,12 +128,16 @@ $btn-hover: #4a99c0;
...
+    .company-name{
+        font-size: 31px !important;
+    }
 }
 
 @media only screen and (max-width:480px) {
 
     .company-name {
-        font-size: 22px;
+        font-size: 22px !important;
```
Comment (MrGDS2) [inline src/components/LandingPage/LandingPage.scss:140]: please explain change to '!important'?

Comment (ksaini75) [reply src/components/LandingPage/LandingPage.scss:140]: I changed it to the code to " !important  " because it was not picking up the the font-size when I change the browser-size on my computer. This corrected the responsiveness for your landing page along with the services page.

File: src/index.js [hunk 3]
```diff
@@ -3,22 +3,21 @@ import ReactDOM from 'react-dom';
 import { BrowserRouter as  Route, Switch } from "react-router-dom";
```
Comment (MrGDS2) [inline src/index.js]: Add Router Here so there's not Error that occurs. 
replace 'Route' with 'Router'

Comment (ksaini75) [reply src/index.js]: I do not understand why I need to change the Route to Router, if the about-page and landing page reflects the same syntax.

Comment (MrGDS2) [reply src/index.js]: add this
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";

Comment (ksaini75) [reply src/index.js]: Okay, but why am I changing it from Route to Router. Please explain.

Comment (MrGDS2) [reply src/index.js]: The merge mistake on my part is the reason I was saying you can either change the "Route" import to "Router" but you can just, change the "Router" that I changed to "Route" inside index.js that would probably be easier than what I asked about before. I apologize for the confusion

**Comment by MrGDS2** [comment]
Had another merge conflict but I fixed it, this is good to merge.
